app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(team: list[Player]) -> int:
+    return sum(player.get_rating() for player in team)
+
+
+def elves_concert(elves_list: list[Elf]) -> None:
+    [elf.play_elf_song() for elf in elves_list]
+
+
+def feast_of_the_dwarves(dwarf_list: list[Dwarf]) -> None:
+    [dwarf.eat_favourite_dish() for dwarf in dwarf_list]
```

Comment:
List comprehension is good for creating lists but for loop is good for call the function

Reply:
Rewrote